frontend/packages/ceph-storage-plugin/integration-tests/tests/1-install/installFlow.scenario.ts

```diff
@@ -134,6 +146,148 @@ if (TEST_PLATFORM === Platform.OCP || (TEST_PLATFORM === Platform.OCS && MODE ==
   });
 }
 
+if (TEST_PLATFORM === Platform.OCS && MODE === Mode.ATTACHED_DEVICES) {
+  describe('Test creation of Attached Storage Cluster', () => {
+    it('Info Alert should be shown as LSO will not be installed', async () => {
+      await Installer.createAttachedStorageCluster();
+      const msg = await currentSelectors.LSOAlert.getText();
+      expect(msg.includes(LSO_INFO_MSG)).toBe(true);
+    });
+
+    it(
+      'tests subscription flow for LSO Operator',
+      async () => {
+        await Installer.subscribeToLSOOperator();
+      },
+      3 * MINUTE,
+    );
+
+    it('Create Storage Class Wizard should be present once LSO is installed and no storage class is present', async () => {
+      await goToInstalledOperators();
+      await Installer.selectOCSOperator();
+      await Installer.createAttachedStorageCluster();
+      expect(currentSelectors.LSOWizard.isPresent()).toBeTruthy();
+    });
+
+    it('Create Storage Cluster View should be present when LSO is installed and storage class is present', async () => {
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc apply -f -`);
+      await goToInstalledOperators();
+      await Installer.selectOCSOperator();
+      await Installer.createAttachedStorageCluster();
+      // wizard should not be present
+      expect(currentSelectors.LSOWizard.isPresent()).toBeFalsy();
+
+      // installation page should not be present
+      const msg = await currentSelectors.LSOAlert.getText();
+      expect(msg.includes(LSO_INFO_MSG)).toBe(false);
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc delete -f -`);
+    });
+
+    it('Should show error message on Create Storage Cluster View, if storage class does not contain minimum 3 nodes ', async () => {
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc apply -f -`);
+      await goToInstalledOperators();
+      await Installer.selectOCSOperator();
+      await Installer.createAttachedStorageCluster();
+      await click(currentSelectors.scDropdown);
+      await click(currentSelectors.selectSC(storageClass.metadata.name));
+      await browser.sleep(40 * SECOND);
+      await browser.wait(until.visibilityOf(currentSelectors.errorAlert));
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      expect(currentSelectors.createNewSCBtn.isPresent()).toBeTruthy();
+      execSync(`echo '${JSON.stringify(storageClass)}' | oc delete -f -`);
+    });
+
+    it('Should be able to create storage class, when no storage class is present for attached devices', async () => {
+      const scName = 'lvs';
+
+      await goToInstalledOperators();
+      await Installer.selectOCSOperator();
+      await Installer.createAttachedStorageCluster();
+
+      // 1st step
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      let step = await currentSelectors.currentStep.getText();
+      expect(step.includes(SC_STEPS_NAME.DISCOVER)).toBeTruthy();
```

Comment:
```suggestion
      expect(step.includes(SC_STEPS_NAME.DISCOVERY)).toBeTruthy();
```
We are using `discovery` in a few places so lets keep it all aligned.